<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchAndReplaceSettings</class>
 <widget class="QWidget" name="SearchAndReplaceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search &amp; Replace Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbSettings">
     <property name="title">
      <string>Actions triggers</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbReplaceSearchText">
        <property name="text">
         <string>Replace search text by selected text.</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" rowspan="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>68</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="cbOnlyWhenNotVisible">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Only when the search widget is not visible.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="cbOnlyWhenNotRegExp">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Only when not using RegExp mode.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="cbOnlyWhenNotEmpty">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Only when the selected text is not empty.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dbbButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbReplaceSearchText</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbOnlyWhenNotVisible</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbReplaceSearchText</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbOnlyWhenNotRegExp</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbReplaceSearchText</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbOnlyWhenNotEmpty</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
 </slots>
</ui>
